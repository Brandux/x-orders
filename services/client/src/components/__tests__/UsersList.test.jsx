import React from 'react';
import { shallow } from 'enzyme';

import UsersList from '../UsersList';

import renderer from 'react-test-renderer';

const users = [
    {
        'active': true,
        'email': 'brandux@upeu.edu.pe',
        'id': 1,
        'username': 'brandux'
    },
    {
        'active': true,
        'email': 'didier@upeu.edu.pe',
        'id': 2,
        'username': 'didier'
    },
]

test('UsersList rendering propely', () => {
    const wrapper = shallow(<UsersList users= {users}/>);
    const element = wrapper.find('h4');
    expect(element.length).toBe(2);
    expect(element.get(0).props.children).toBe('brandux');
});

test('UsersList renders a snapshop propely', () => {
    const tree = renderer.create(<UsersList users = {users}/>).toJSON();
    expect(tree).toMatchSnapshot();
});
