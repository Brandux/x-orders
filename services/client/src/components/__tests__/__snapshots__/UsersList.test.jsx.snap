// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders a snapshop propely 1`] = `
<div>
  <h4
    className="box title is-4"
  >
    brandux
  </h4>
  <h4
    className="box title is-4"
  >
    didier
  </h4>
</div>
`;
